core/src/main/java/com/orange/ouds/core/component/OudsTag.kt

```diff
@@ -159,22 +159,27 @@ fun OudsTag(
                 if (hasAsset) {
                     Box(
                         modifier = Modifier
-                            .size(assetSize(size))
                             .semantics { hideFromAccessibility() }
                     ) {
                         if (hasLoader) {
                             ProgressIndicator(status = status, appearance = appearance, size = size, progress = loader.progress, enabled = enabled)
                         } else {
                             val isBulletIcon = status.icon is OudsTagIcon.Bullet
```

Comment:
The bullet and non functional icons are also scaled. I can't find a clear statement on this, do you know if it is what we want?